photo-helper/src/main/java/io/intrepid/photohelper/PhotoHelper.java

```diff
@@ -157,8 +202,9 @@ void showImagePickerIntent() {
             view.getPhotoPresenter().logError(view.getIOExceptionMessage(), e);
         }
 
-        final Intent chooserIntent = Intent.createChooser(galleryIntent, fragment.getString(view.getChooserLabel()));
+        final Intent chooserIntent = Intent.createChooser(galleryIntent, contextDelegate.getContext()
+                .getString(view.getChooserLabel()));
```

Comment:
Formatting - can you move the contextDelegate.getContext() onto this line?  Otherwise, it looks like getString() might be called on Intent.createChooser()